### PR TITLE
dark icons - small chat changes

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] name icons `iconDark-whatever` and run the script and it'll support dark mode

### CODE REVIEW

**Comment by chrisnojima** [comment]
cc: @adamjspooner

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/input-area/normal/platform-input.desktop.tsx [hunk 1]
```diff
@@ -452,7 +452,7 @@ const styles = Styles.styleSheetCreate(
         textAlign: 'left',
       },
       inputEditing: {
-        color: Styles.globalColors.blackOrWhite,
+        color: Styles.globalColors.blackOrBlack,
```
Comment (jakob223) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.tsx]: 🎉

File: shared/common-adapters/icon.desktop.tsx [hunk 2]
```diff
@@ -163,13 +163,14 @@ class Icon extends Component<Props, void> {
   }
 }
 
-const imgName = (type: IconType, ext: string, mult: number, prefix?: string, postfix?: string) =>
```
Comment (jakob223) [inline shared/common-adapters/icon.desktop.tsx]: intentionally removing typing for icon names?

Comment (chrisnojima) [reply shared/common-adapters/icon.desktop.tsx]: yeah its fine. its not exported and only called internally by iconTypeToSrcSet

**Pull request merged**